synapse/tools/storm.py

```diff
@@ -402,16 +402,20 @@ async def main(argv, outp=s_output.stdout):
 
             async with await StormCli.anit(proxy, outp=outp, opts=opts) as cli:
 
+                stormopts = {}
+                if opts.view:
+                    stormopts["view"] = opts.view
+
                 if opts.onecmd:
-                    await cli.runCmdLine(opts.onecmd)
+                    await cli.runCmdLine(opts.onecmd, opts=stormopts)
```

Comment:
The StormCLI object has a `opts` argument which is currently unprocessed. We could process that to update self.stormopts and I believe we would not have to updated the base CLI class?